python/ray/serve/_private/deployment_graph_build.py

```diff
@@ -16,13 +15,8 @@
     DeploymentFunctionExecutorNode,
 )
 from ray.serve._private.deployment_function_node import DeploymentFunctionNode
```

Comment:
It's also not clear why we need to have a separate `DeploymentFunctionNode` anymore -- can we unify all of these to one node type?